cves/kernel/CVE-2013-3224.yml

```diff
@@ -221,8 +242,8 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: net/bluetooth
```

Comment:
["net","bluetooth"] is probably better as the subsystem to separate it into two.

Reply:
thanks. I updated it.